common/src/chain/upgrades/chainstate_upgrade.rs

```diff
@@ -25,30 +31,69 @@ pub enum RewardDistributionVersion {
     V1,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Ord, PartialOrd)]
+pub enum TokensFeeVersionVersion {
+    /// Initial tokens fee values
+    V0,
+    /// Updated tokens fee values
+    V1,
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Ord, PartialOrd)]
+pub enum TokensTickerMaxLengthVersion {
+    /// Initial tokens ticker max length values
+    V0,
+    /// Updated tokens ticker max length values
+    V1,
+}
+
 #[derive(Debug, Clone, PartialEq, Eq, Ord, PartialOrd)]
 pub struct ChainstateUpgrade {
```

Comment:
tbh this is getting ridiculous with all these versions. I'd rework this struct into an enum of continuous forks, so that right now it would be an enum of 2. Thoughts?

Reply:
You can do that (in another PR). Though keep in mind that you cannot make the upgrades monotonic (like we tried to do before) because that will make it some cases mutually exclude others. Like for example, you cannot run PoW tests and have them upgraded.